google-cloud-clients/google-cloud-bigquery/src/test/java/com/google/cloud/bigquery/it/ITBigQueryTest.java

```diff
@@ -1243,6 +1262,26 @@ public void testListJobsWithSelectedFields() {
     }
   }
 
+  @Test
+  public void testListJobsWithCreationBounding() {
+    long currentMillis = currentTimeMillis();
+    long lowerBound = currentMillis - 3600 * 1000;
+    long upperBound = currentMillis;
+    Page<Job> jobs =
```

Comment:
Should we be checking that we are actually getting results back?

Reply:
added an assertion that the jobcount is nonzero.